app/views/partials/code.scala.html

```diff
@@ -55,9 +53,7 @@ <h3 class="board__heading">Code</h3>
                 @commenterReport.map { report =>
                     <li class="list-item"><a id="link-to-@{repo.name}-comments" href="@prcommenterRoutes.PrCommenterController.recommendations(name = Some(report.name))">PR-Commenter recommendations (@report.comments.size)</a></li>
                 }
-                @if(CatalogueFrontendSwitches.surfacingVulnerabilities.isEnabled & repo.repoType == RepoType.Service) {
-                    <li class="list-item"><a id="link-to-vulnerabilities" href="@uk.gov.hmrc.cataloguefrontend.vulnerabilities.routes.VulnerabilitiesController.distinctVulnerabilitySummaries(None, curationStatus = Some(CurationStatus.ActionRequired.asString), Some(s"\"${repo.name}\""), None)">Vulnerabilities (@distinctVulnerabilitiesCount)</a></li>
-                }
+                <li class="list-item"><a id="link-to-vulnerabilities" href="@uk.gov.hmrc.cataloguefrontend.vulnerabilities.routes.VulnerabilitiesController.distinctVulnerabilitySummaries(None, curationStatus = Some(CurationStatus.ActionRequired.asString), Some(s"\"${repo.name}\""), None)">Vulnerabilities (@distinctVulnerabilitiesCount)</a></li>
```

Comment:
still need to check it's a Service? Can we roll all the service checks into one?

Reply:
Do'h trigger happy

Reply:
Done and merged into check below it.